components/pedal-details-main/index.js

```diff
@@ -1,58 +1,58 @@
 import React from "react";
 import Image from "next/image";
 import {
-  CategoryWrapper,
-  ImageWrapper,
-  Tag,
-  TagContainer,
+  StyledCategoryWrapper,
+  StyledImageWrapper,
+  StyledTag,
+  StyledTagContainer,
 } from "../pedal-card-preview/pedal-card-preview.styled";
 import {
   StyledBypass,
   StyledComment,
   StyledCommentField,
   StyledCommentLegend,
   StyledContentMain,
-  StyledLegend,
   StyledMono,
   StyledPedalManufacturerDetail,
   StyledTipp,
   StyledTippField,
+  StyledTippLegend,
 } from "./pedal-main.styled";
-export default function PedalMainPage({ pedalData }) {
+
+export default function PedalMainPage({ currentPedal }) {
   return (
     <StyledContentMain>
       <StyledPedalManufacturerDetail>
-        {pedalData.manufacturer}, {pedalData.made_in}
+        {currentPedal.manufacturer}, {currentPedal.made_in}
       </StyledPedalManufacturerDetail>
-      <CategoryWrapper>
-        {pedalData.category.map((category) => (
+      <StyledCategoryWrapper>
+        {currentPedal.category.map((category) => (
```

Comment:
same here - i think this `p` should be an `li`

Reply:
here i agree